test/selenium/lib/foxboxHelperSelenium.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
Yeah, that could be a hassle to modify the helper coming from the integration tests. Let's keep it as is, and factorize them in a next PR